splink/comparison_level_library.py

```diff
@@ -161,12 +168,13 @@ def __init__(
                     "transforms applied to it such as lower(), "
                     "substr() etc."
                 )
-
-            config["tf_adjustment_column"] = col_name
-            config["tf_adjustment_weight"] = 1.0
             # leave tf_minimum_u_value as None
+            self.configure(
+                tf_adjustment_column=self.col_expression,
```

Comment:
The problem is that here, the tf_adjustment_column shold possibly/probably (?) be set to a string value (because configure directly configures the actual dict).  

Or is that the wrong way of thinking about it and .configure needs some indirection